CONTRIBUTING.md

```diff
@@ -145,9 +144,11 @@ documentation in HTML format.
 To build the documentation locally:
 
 ```bash
-$ make docs
+  $ poetry run make html --directory docs
```

Comment:
How does this work if the makefile doesn't exist?

Reply:
ohh right, poetry is running make, however all of the dependencies are provided by poetry (in pyproject.toml) so the make tool is being used however the makefile is not used therefore it's not needed

Reply:
ah. Ok, that makes more sense.